theia-rust-docker/README.md

```diff
@@ -0,0 +1,41 @@
+# Theia with Rust support
+
+## Building the image
+
+RUST\_CHANNEL=nightly make image
+
+## Start Theia
+
+1. cd to a folder in your home dir.
+
+eg. cd to the example\_workspace folder.
+  
+2. Start editor
+
+<pre>[path to theia_apps/theia-rust-docket]/ex_theia</pre>
```

Comment:
typo, I think: docket -> docker? Same repeats a few time in this file

Reply:
I thought that the format was a bit confusing here and in other similar occurrences in this file. Maybe the following would be clearer? 
`[path to theia-apps]/theia-rust-docker/ex_theia`

Reply:
Changed